### PR TITLE
[Bug] All-friends page without text&img "no friends" when user doesn't has any friends /bug #2344

### PR DESCRIPTION
**Pull request by yuriiLisunov**
**Description:**
**BEFORE:**
When user doesn't has any friends, page is empty.
**AFTER:**
When user doesn't has any friends, on page 'all friends' we can see image and text "You don't have any friends".
friendsNO

### CODE REVIEW

**Review by idzhersh — APPROVED** [review]

File: src/app/component/user/components/profile/users-friends/friend-dashboard/all-friends/all-friends.component.scss [hunk 1]
```diff
@@ -11,3 +11,18 @@
...
 }
+
+.noFriends-wraper {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+
+  .noFriends {
+    font-family: 'Montserrat', sans-serif;
```
Comment (JimHolman1199) [inline src/app/component/user/components/profile/users-friends/friend-dashboard/all-friends/all-friends.component.scss:22]: do not hardcode font family, use a global variable or create one

File: src/app/component/user/components/profile/users-friends/friend-dashboard/all-friends/all-friends.component.html [hunk 2]
```diff
@@ -1,4 +1,7 @@
-<div *ngIf="friends && friends.length>0; else spinner">
+<div *ngIf="!friends">
+  <app-spinner></app-spinner>
+</div>
+<div *ngIf="friends && friends.length>0">
```
Comment (JimHolman1199) [inline src/app/component/user/components/profile/users-friends/friend-dashboard/all-friends/all-friends.component.html:4]: not correct formating
expected to be : <div *ngIf="friends && friends.length > 0">

File: src/app/component/user/components/profile/users-friends/friend-dashboard/all-friends/all-friends.component.scss (hunk 1 above)
Comment (yuriiLisunov) [reply src/app/component/user/components/profile/users-friends/friend-dashboard/all-friends/all-friends.component.scss]: fixed

**Comment by yuriiLisunov** [comment]
> image
> view in 320 resolution

first of all: we don't have mockups for small resolutions. Also on resolution <768px My eco friends doesn't imagine (as on mockups)

**Review by VitalyBorysenko — APPROVED** [review]

File: src/app/component/user/components/profile/users-friends/friend-dashboard/all-friends/all-friends.component.html [hunk 3]
```diff
@@ -16,7 +19,13 @@
...
+<div *ngIf="friends && friends.length === 0">
+  <div class="noFriends-wraper">
+    <div class="img-absent">
+      <img [src] = "absent" alt="absent-friends">
+    </div>
+    <div>
+      <p class="noFriends">{{'profile.zero-friends' | translate}}</p>
+    </div>
+  </div>
+</div>
```
Comment (JimHolman1199) [inline src/app/component/user/components/profile/users-friends/friend-dashboard/all-friends/all-friends.component.html:31]: add a new line at the end of file

Comment (yuriiLisunov) [reply src/app/component/user/components/profile/users-friends/friend-dashboard/all-friends/all-friends.component.html]: added

**Review by AlekOK — APPROVED** [review]

**Review by mr-pavlenko — APPROVED** [review]

File: src/app/component/user/components/profile/users-friends/friend-dashboard/all-friends/all-friends.component.html [hunk 4]
```diff
@@ -1,4 +1,7 @@
-<div *ngIf="friends && friends.length>0; else spinner">
+<div *ngIf="!friends">
+  <app-spinner></app-spinner>
+</div>
+<div *ngIf="friends && friends.length > 0">
```
Comment (mr-pavlenko) [inline src/app/component/user/components/profile/users-friends/friend-dashboard/all-friends/all-friends.component.html:4]: you can check by the next way ` *ngIf="friends?.length"`

Comment (yuriiLisunov) [reply src/app/component/user/components/profile/users-friends/friend-dashboard/all-friends/all-friends.component.html]: changed

**Review by JimHolman1199 — APPROVED** [review]

File: src/app/component/user/components/profile/users-friends/friend-dashboard/all-friends/all-friends.component.scss [hunk 5]
```diff
@@ -1,3 +1,5 @@
+$fontMontserrat: 'Montserrat', sans-serif;
```
Comment (JimHolman1199) [inline src/app/component/user/components/profile/users-friends/friend-dashboard/all-friends/all-friends.component.scss:1]: do we have this font in typography folder/ file ?

Comment (yuriiLisunov) [reply src/app/component/user/components/profile/users-friends/friend-dashboard/all-friends/all-friends.component.scss]: yes, thanks. i've changed it

**Review by paslavskyi9roman — APPROVED** [review]

**Pull request merged**